sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/response/IdealIssuersResponse.java

```diff
@@ -0,0 +1,61 @@
+package nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.response;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Response from the Rabobank API when request the list of available issuers.
+ */
+public class IdealIssuersResponse {
+    public static final String ISSUERS_KEY = "issuers";
```

Comment:
can this be made private?

Reply:
This was deliberately left public because I will be reusing the key value for reading/creating the 'issuers' in the json object.